JulioC-19/LargeProjectMobile
# Penny/profile change restriction

Implemented edit profile restriction.

To test:
Login and navigate to the profile screen.
Change names and password and hit edit profile, error message appears accordingly.
